### DISCUSSION TITLE
Trails follow an object path animation?

### DISCUSSION THREAD

**Discussion by chengenzhao** [AlmasB/FXGL#1321] [Q&A]
Hi:

Is it any way to create a trails following an object animation on a path?
like this https://www.youtube.com/watch?v=FreApl7CXL4&ab_channel=andrewpach%E2%80%A2PowerPointandvideo

thx

**Comment by AlmasB** [comment]
@chengenzhao Hi, given the first (main) object's position values are known, I suppose it's just a matter of capturing these values into a `List<Point2D>`. Then just apply these values to the second object with a delay, making it follow the main object.
